sirepo/pkcli/test_http.py

```diff
@@ -328,10 +306,18 @@ async def post(self, uri, data, caller):
                         'Content-type',  'application/json'
                     ),
                     method='POST',
-                    **self._FETCH_DEFAULT_ARGS
+                    **self._fetch_default_args
                 ),
             )
 
+    @property
```

Comment:
@robnagler this is wonky but I wasn't sure how else to implement this. If we define _fetch_default_args as a constant on the class like:
```python
class _Client:
  _FETCH_DEFAULT_ARGS = PKDict(..., validate_cert=cfg.validate_cert)
```

I get an error `nonetype has no attribute validate_cert`. This is because cfg is None when this code is evaluated. Do you have a better way of implementing this?

Reply:
I wouldn't bother with making it a property. That's an optimization. :)  I would just call the function fetch_default_args. It doesn't need to be a constant.

Reply:
:)

https://github.com/radiasoft/sirepo/pull/2752/commits/355420a7e87bf9b04bf43496ba6731e81ed70d2d